.github/workflows/build-unified.yml

```diff
@@ -49,7 +47,7 @@ jobs:
         run: |
           IFS="," read -a publish_branches <<< $PUBLISH_BRANCHES
 
-          if [[ " ${publish_branches[*]} " =~ " ${GITHUB_REF:11} " && " ${publish_branches[*]} " =~ " ${SERVER_BRANCH} " ]]; then
```

Comment:
This was a protec tion from running the workflow from a feature branch in `self-host` and `master`/`rc`/`hotfix-rc` branch from server and publishing that as a `dev` tag.

Reply:
So if I understand correctly, we want to prevent a user from manually triggering the `build-unified` workflow on a feature branch in the `self-host` repository while targeting one of the publish branches in the `server` repository?

Reply:
Exactly 😃